grid-proxy/internal/explorer/state.go

```diff
@@ -0,0 +1,87 @@
+package explorer
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/types"
+	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go"
+)
+
+const (
+	OkState = "ok"
+)
+
+func createReport(db DBClient, peer rmb.Client) types.Healthiness {
+	var report types.Healthiness
+
+	// db connection
+	report.DBConn = OkState
+	err := db.DB.Ping()
+	if err != nil {
+		report.DBConn = err.Error()
+	}
```

Comment:
making it in one statement looks more clean imo  like what u did with pingRandomTwins underneath 
```
if err := db.Ping(); err != nil {
		report.DBConn = err.Error()
	}
```